provisioners/configure.sh

```diff
@@ -114,7 +114,7 @@ envsubst < $TEMPLATES_PATH/etc/permanent/notification-service.env > /etc/permane
 systemctl enable notification.service
 
 echo "Install node global packages"
-npm install npm --global
+npm install npm@8.17.0 --global
```

Comment:
This looks good.  I'm curious if we have a sense why we're bothering to install a different version of npm in the first place.

Reply:
I don't have a clear idea of why we're doing that. The version installed by default (6.14.16) is a few major versions behind current, but we're also using a version of node that's several major versions back.